data-model/SDP-sample-records/script_sample1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/script/1",
+  "label": "Serto (script)",
+  "entity_type": "script",
+  "uniform_name": "Serto (script)",
+  "description": "Serto is a cursive Syriac script that developed throughout the 7th century (CE) and became the dominant script in West Syrian manuscripts.",
+  "authority_files": [],
```

Comment:
as with writing system, do we have any external authority files or LOD vocabularies defining scripts?